### ISSUE TITLE
Is there a way to order with aggregated column?

### ISSUE THREAD

**Issue by devsarvesh92** [piccolo-orm/piccolo#669]
Hi,

I want to order with aggregated column e.g 'revenue_agg', is there a way to do it?

Facility.select(
            Facility.code, Facility.name, Sum(Facility.revenue).as_alias("revenue_agg")).where(
            Facility.practice.name == practice_name.title().strip(),
            Facility.year == year,
        ).group_by(Facility.practice.name, Facility.year, Facility.name, Facility.code).order_by(
            Facility.code,
            "revenue_agg",
            ascending=False,
        )

**Comment by dantownsend** [comment]
Not at the moment, but we're working on a solution where you can order by anything.

In the mean time you can work around it like this:

```python
query = Facility.select(
    Facility.code,
    Facility.name,
    Sum(Facility.revenue).as_alias("revenue_agg")
).where(
    Facility.practice.name == practice_name.title().strip(),
    Facility.year == year,
).group_by(
    Facility.practice.name, Facility.year, Facility.name, Facility.code
)

await Facility.raw(query.querystrings[0].__str__() + ' ORDER BY revenue_agg')
```

**Comment by devsarvesh92** [comment]
Ok thanks @dantownsend

**Comment by dantownsend** [comment]
Fixed in [0.97.0](https://github.com/piccolo-orm/piccolo/releases/tag/0.97.0).
